db/seeds.rb

```diff
@@ -56,55 +56,6 @@ def fetch!
   end
 end
 
-tags = [
```

Comment:
I've removed the random tag seeding as the tags are now synced in the `SyncTrack` command, and would thus be overwritten later.